cfgov/jinja2/v1/learn-page/index.html

```diff
@@ -4,7 +4,6 @@
 {% import 'molecules/related-metadata.html' as related_metadata with context %}
 
 {% set parent = page.parent() %}
-{% set breadcrumb_items = [(parent.url, parent.url, parent.title)] %}
```

Comment:
lets also remove this line from the `browse-basic`, `browse-filterable`, `document-detail` templates